integration-tests/pom.xml

```
@@ -340,6 +340,20 @@
             </modules>
         </profile>
 
+        <!-- Java 17 test profile that restricts running in 17+ JVM version -->
+        <profile>
+            <id>java-17</id>
+            <activation>
+                <property>
+                    <name>!no-test-modules</name>
+                </property>
+                <jdk>(17,)</jdk>
```

I would have expected:
```suggestion
                <jdk>[17,)</jdk>
```
Why did you use a `(`?

---

@gsmet because ... I made a mistake ;)
Thanks for pointing this out, I'll update it.